### DISCUSSION TITLE
My Copilot doesn't work due to a billing issue

### DISCUSSION THREAD

**Discussion by VictorAtencio-21** [community/community#40097] [Projects and Issues]
### Select Topic Area

Bug

### Body

Hello!,

It seems that my copilot stopped working, for some reason it tells me that I owe money, but I have the Pro plan provided by GitHub students. Before that I was using copilot through the free trial, but I cancelled it once I joined GitHub students.

Is there anything I can do?, I don't know where the money I'm supposed to owe came from.

Thank you very much for your help!

**Comment by ossanna16** [comment]
Hi there @VictorAtencio-21 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
